docs/nodes/configuration.md

```diff
@@ -409,16 +471,22 @@ peer-query-maj23-sleep-duration = "2s"
 #######################################################
 [tx-index]
 
-# What indexer to use for transactions
+# The backend database list to back the indexer.
+# If list contains null, meaning no indexer service will be used.
 #
 # The application will set which txs to index. In some cases a node operator will be able
 # to decide which txs to index based on configuration set in the application.
 #
 # Options:
 #   1) "null"
 #   2) "kv" (default) - the simplest possible indexer, backed by key-value storage (defaults to levelDB; see DBBackend).
-#   - When "kv" is chosen "tx.height" and "tx.hash" will always be indexed.
```

Comment:
(ignorance on my part) Is this not true in postgres mode?

Reply:
you are right, it is true for both scenarios